certbot/CHANGELOG.md

```diff
@@ -16,7 +16,9 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 
 ### Fixed
 
-*
+* The `show_account` now uses the "newAccount" ACME endpoint to fetch the account data, so it
```

Comment:
New Certbot has been released, please merge `master` and move this entry to the new section.